src/encoded/static/components/genome_browser.js

```diff
@@ -0,0 +1,432 @@
+import React from 'react';
+
+
+const maxFilesBrowsed = 5; // Maximum number of files to browse
```

Comment:
I don't like this that low.   Maybe it could guess based on vertical size of screen